app/org/sagebionetworks/bridge/util/DuplicateZipEntryException.java

```diff
@@ -0,0 +1,13 @@
+package org.sagebionetworks.bridge.util;
```

Comment:
This should be fine, since we never return these exceptions directly to the user. (We could enforce catching them by making these checked exceptions.)